s3/replication/manager/tests/system/load_transfer_test.py

```diff
@@ -261,49 +261,59 @@ async def run_load_test():
     polling_count = test_config.polling_count
 
     while jobs_running and polling_count != 0:
-        job_status_task_list = []
-        for job_id in posted_jobs_set:
-            logger.debug(
-                "Checking status for job: job_id= {}".format(job_id))
-
-            job_status_task = asyncio.ensure_future(manager_session.get(
-                url + '/jobs/' + job_id))
-            job_status_task_list.append(job_status_task)
-
-        # jobs status contains list of response from GET /jobs/<job_id> for
-        # each
-        logger.debug("Waiting for all GET {}/jobs/<job_id> response...".
-                     format(url))
-        get_job_response_list = await asyncio.gather(*job_status_task_list)
-        # logger.debug("get_job_response_list: {}".format(
-        #     get_job_response_list))
-
-        for get_job_resp in get_job_response_list:
-            job_status = await get_job_resp.json()
-            job_id = job_status["job"]["job_id"]
-            job_state = job_status["job"]["state"]
-            logger.debug(
-                "job_state = {} for job_id {}".format(
-                    job_state, job_id))
-            if job_state == "COMPLETED":
-                # remove completed job from polling list
-                posted_jobs_set.remove(job_id)
-
-        if len(posted_jobs_set) == 0:
-            # No jobs pending.
+        async with manager_session.get(url + '/jobs?count&completed') as resp:
+            logger.info(
+                'GET jobs returned http Status: {}'.format(resp.status))
+            response = await resp.json()
+
+        completed_count = response['count']
```

Comment:
should this be inside async with block? scope of `response` changes

Reply:
done